tutorials/shaders/compute_shaders.rst

```diff
@@ -189,18 +283,22 @@ The steps we need to do to compute our result are:
     rd.ComputeListDispatch(computeList, xGroups: 5, yGroups: 1, zGroups: 1);
     rd.ComputeListEnd();
 
-Note that we are dispatching the compute shader with 5 work groups in the x-axis, and one in the others.
-Since we have 2 local invocations in the x-axis (specified in our shader) 10 compute shader invocations will be launched in total.
-If you read or write to indices outside of the range of your buffer, you may access memory outside of your shaders control or parts of other variables which may cause issues on some hardware.
-
+Note that we are dispatching the compute shader with 5 work groups in the
+x-axis, and one in the others. Since we have 2 local invocations in the x-axis
+(specified in our shader) 10 compute shader invocations will be launched in
+total. If you read or write to indices outside of the range of your buffer, you
+may access memory outside of your shaders control or parts of other variables
+which may cause issues on some hardware.
```

Comment:
```suggestion
Note that we are dispatching the compute shader with 5 work groups in the
X axis, and one in the others. Since we have 2 local invocations in the X axis
(specified in our shader), 10 compute shader invocations will be launched in
total. If you read or write to indices outside of the range of your buffer, you
may access memory outside of your shaders control or parts of other variables
which may cause issues on some hardware.
```